soc/realtek/ec/rts5912/Kconfig.soc

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+#
+
+config SOC_SERIES_RTS5912
+	bool
+	select SOC_FAMILY_REALTEK_EC
+	help
+	  Enable support for REALTEK EC MCU series
+
+config SOC_SERIES
+	default "rts5912" if SOC_SERIES_RTS5912
+	
+config SOC_RTS5912
+	bool
+	select SOC_SERIES_RTS5912
+	
+config SOC
+	default "rts5912" if SOC_RTS5912
```

Comment:
non-blocking comment: do you really need a soc series when the soc and soc series name as the same? Is this really a series?

Reply:
There may be series and plenty of SoCs with different names in the future.

In order to maintain the consistency of the folder structure, we will use the same name for the series and SoCs for now.